packages/adapter-cloudflare/src/worker.js

```diff
@@ -28,7 +36,7 @@ const worker = {
 
 		const stripped_pathname = pathname.replace(/\/$/, '');
 
-		// prerendered pages and /static files
+		// /static files, the service worker, and Vite imported server assets
```

Comment:
the `/` right after `//` is a little tricky to read
```suggestion
		// files in /static, the service worker, and Vite imported server assets
```